deploy/olm-catalog/service-assurance-operator/0.1.0/service-assurance-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -12,7 +12,8 @@ metadata:
           },
           "spec": {
             "eventsEnabled": false,
-            "metricsEnabled": true
+            "metricsEnabled": true,
```

Comment:
Don't modify this manually. You use the `operator-sdk` to build this out using `gen-csv`

Reply:
Also note that you want to use operator-sdk 0.12.0 or 0.13.1 if it is available. The operator-sdk 0.13.0 has a bug in it which destroys the CSV values under the `customresourcedefinition.owned` section when using a non-Go Operator (like we're doing here).

When you're ready for the `olm-catalog` directory to be updated let me know, and I'm happy to generate the updated package manifest as a commit here. I can also run the scorecard and all that as well. Happy to give you a demo when you are ready.

Reply:
I was seeing some weird behaviour with operator-sdk, wondering why the contents of that file were obliterated; it was version 0.13.0! I'll re-install a different version and prod it. 

Would appreciate a demo tomorrow!

Reply:
No problem, my calendar is pretty open, so just ping me when you're around on chat and I can show you what I remember from before the break :) Remind me to record it and I'll ship it off to @csibbitt who is off tomorrow.